### ISSUE TITLE
Sponsor > New tab in the admin doesn't have a field for the logo image. 

### ISSUE THREAD

**Issue by arch-jslin** [TGDF/official-site#466]
Only broken in 2022 site.

**Comment by arch-jslin** [comment]
Also, I can't create a new sponsor at all. The 新增Sponsor button currently doesn't do anything.

**Comment by elct9620** [comment]
Does the logo is attached?

**Comment by arch-jslin** [comment]
Updating this issue with solution, so we won't be confused by it again: 

The key word "sponsor" is being blocked by ad-blocking browser extensions. 

I am leaving this open so it's easier to see, but I am moving this to low priority.
